nss-day-cohort-30/bangazon-site-billowy-beluga
# Added basic controllers and views

# Description
Used VS templates to create controllers and views for Product, ProductType, PaymentType, and Order

## Type of change
Please delete options that are not relevant.

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout branchdressing`
3. start BangazonSite.sln
4. Verify controllers and views exist


# Checklist:
- [ X] I have added test instructions that prove my fix is effective or that my feature works
